src/lib/FreeC/Pass/DefineDeclPass.hs

```diff
@@ -24,19 +24,14 @@
 --   = Error cases
 --
 --   * The user is informed if a different name is assigned to an entry.
+module FreeC.Pass.DefineDeclPass ( defineTypeDeclsPass, defineFuncDeclsPass ) where
```

Comment:
This line is 84 characters long, this looks like a bug in Floskell to me.

Reply:
That's not really a bug. Floskell does not have a hard line limit but a penalty function. Seems like I have to increase the penalty for overfull lines or reduce the penalty for line breaks.

Reply:
After playing around with the penalties a bit more, it seems like they are not at fault in this case. It seems like Floskell does not consider a line overfull if the `where` keyword starts before the 80 character limit.

Reply:
That's what I thought to cause this long line. I would consider that to be a bug.